molgenis-beacon/src/main/java/org/molgenis/beacon/service/BeaconServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.beacon.service;
+
+import org.molgenis.data.DataService;
+import org.molgenis.data.support.QueryImpl;
+import org.molgenis.data.vcf.model.VcfAttributes;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static java.util.Objects.requireNonNull;
+
+@Component
+public class BeaconServiceImpl implements BeaconService
+{
+	private DataService dataService;
+
+	public BeaconServiceImpl(DataService dataService)
+	{
+		this.dataService = requireNonNull(dataService);
+	}
+
+	@Override
+	public boolean query(String chromosome, Long position, String reference, String allele, String entityTypeID)
+	{
+		/* Use the count() method to determine if a variation exists */
+		return dataService.count(entityTypeID, new QueryImpl<>().eq(VcfAttributes.CHROM, chromosome)
```

Comment:
Have you considered using sys_genomebrowser_GenomeBrowserAttributes to determine which attributes to query?